### DISCUSSION TITLE
Still in development?

### DISCUSSION THREAD

**Discussion by cullenohara** [bitbrain/pandora#200] [Q&A] [answered]
As stated in the title, wondering if this is still under development?

**Comment by bitbrain** [comment, marked as answer]
@cullenohara Thanks for your interest in the project! This Godot addon is very much alive and under development. Like with many open-source projects, the beauty lies in collaboration. Development is always ongoing, shaped by [whichever issues come up or are important](https://github.com/bitbrain/pandora/issues).

Even though it might sometimes slow down, it's an evolving project, and I encourage anyone interested to contribute, share ideas, or report issues. Open source is all about growth through collaboration, and I'm excited to see how this addon continues to develop with the help of people like you!

Feel free to jump in or ask any more questions!
